docs/wiki/[Examples]-Deploy-Management-Resources.md

```diff
@@ -0,0 +1,50 @@
+## Overview
+This example will deploy resources for centrally managing the Enterprise Scale Landing Zone. This module will deploy the following resources with default configurations that can be adjusted once you are up and going in the subscription of your specification:
+
+
+* resource group
+* log analytics workspace
+* automation account
+* linked service to the analytics automation account
+
+
+
+The module then goes onto configure Deploy if Not Exist policies to enable features within Log Analytics and Sentinel including
```

Comment:
Let discuss this in more detail so we can provide a full breakdown of what this module actually does in terms of the policies.

At a very high-level, the module simply updates input parameter values for existing Policy Assignments included at various scopes within a default implementation of ES, and toggles the enforcement mode.

For consistency, we should probably also use the format "DeployIfNotExists" instead of "Deploy if Not Exist".